packages/babel-traverse/src/path/family.js

```diff
@@ -67,6 +67,30 @@ export function getSibling(key) {
   });
 }
 
+export function getPrevSibling() {
+  return this.getSibling(this.key - 1);
+}
+
+export function getNextSibling() {
+  return this.getSibling(this.key + 1);
+}
+
+export function forEachNextSibling(callback = Function.prototype) {
```

Comment:
@chitchu Do you have a specific usecase in mind for this? If we add one like this, why not `.map` or `.filter` or any number of other array methods? I'd probably be more in favor of a `getAllNextSiblings` that returned an array, rather than re-implementing array-like behavior.